soundfile.py

```diff
@@ -695,6 +713,10 @@ def __init__(self, file, mode='r', samplerate=None, channels=None,
     """Whether the sound file is closed or not."""
     _errorcode = property(lambda self: _snd.sf_error(self._file))
     """A pending sndfile error code."""
+    compression_level = property(lambda self: self._compression_level)
```

Comment:
These properties probably can't be safely modified at runtime. They should therefore be read-only properties.

Reply:
I think these properties are already read-only.
The following code throw an exception.
```python
with SoundFile(...) as f:
    f.compression_level = 1
> AttributeError: property 'compression_level' of 'SoundFile' object has no setter
```

Reply:
My bad, sorry.